### PR TITLE
Symfony 6

### LINKED ISSUE #59: Symfony 6 support ?
Is there a plan for Symfony 6 support ?

### PR DESCRIPTION
**Pull request by Imagica**
Have bumped up some versions for Symfony 6. Resolves #59

### CODE REVIEW

**Comment by Imagica** [comment]
Any chance this can be updated? Symfony 6 has been out for a while now.

**Comment by welcoMattic** [comment]
Yes, but your last commit breaks the CI: https://github.com/php-translation/loco-adapter/runs/5229013974?check_suite_focus=true

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /phpspec.yml
 /phpunit.xml
 /vendor/
+/.idea
```
Comment (welcoMattic) [inline .gitignore:6]: This one must be in *your* local gitignore, not in the project one

**Comment by welcoMattic** [comment]
CI is still failing, could you take a look to fix it? Thanks a lot, I will merge as soon as it will be green

**Comment by welcoMattic** [comment]
Replaced by #61

**Comment by welcoMattic** [comment]
Symfony 6 support is released. https://github.com/php-translation/loco-adapter/releases/tag/0.11.2

**Pull request closed without merging**